core/server/api/v2/configuration.js

```diff
@@ -0,0 +1,67 @@
+const Promise = require('bluebird');
+const {isPlainObject} = require('lodash');
+const urlService = require('../../services/url');
+const models = require('../../models');
+const config = require('../../config');
+const labs = require('../../services/labs');
+const settingsCache = require('../../services/settings/cache');
+const ghostVersion = require('../../lib/ghost-version');
+
+function fetchAvailableTimezones() {
+    const timezones = require('../../data/timezones.json');
+    return timezones;
+}
+
+function getAboutConfig() {
+    return {
+        version: ghostVersion.full,
+        environment: config.get('env'),
+        database: config.get('database').client,
+        mail: isPlainObject(config.get('mail')) ? config.get('mail').transport : ''
+    };
+}
+
+function getBaseConfig() {
+    return {
+        useGravatar: !config.isPrivacyDisabled('useGravatar'),
+        publicAPI: labs.isSet('publicAPI'),
```

Comment:
The admin client still uses the labs checkbox for the `publicAPI`. So we still need to serve this config 👍